scylla-cql/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "scylla-cql"
+version = "0.0.1"
+edition = "2021"
+description = "CQL data types and primitives, for interacting with Scylla."
+repository = "https://github.com/scylladb/scylla-rust-driver"
+readme = "../README.md"
+keywords = ["database", "scylla", "cql", "cassandra"]
+categories = ["database"]
+license = "MIT OR Apache-2.0"
+
+[dependencies]
+scylla-macros = { version = "0.1.1", path = "../scylla-macros"}
+byteorder = "1.3.4"
+bytes = "1.0.1"
+num_enum = "0.5"
+tokio = { version = "1.12", features = ["net", "time", "io-util", "sync", "rt", "macros"] }
```

Comment:
Please remove the features from the list. I checked and we only use `AsyncRead`, `AsyncReadExt` and they don't need any of the tokio features enabled.

Just

```toml
tokio = "1.12"
```

should suffice.